lib/evss/auth_headers.rb

```diff
@@ -37,11 +37,23 @@ def eauth_json
           edi: @user.edipi,
           firstName: @user.first_name,
           lastName: @user.last_name,
-          birthDate: iso8601_birth_date
+          birthDate: iso8601_birth_date,
+          gender: gender
```

Comment:
Has EVSS confirmed that passing `gender` for all our calls to EVSS will be OK? I don't _think_ it's an issue, just keep an eye on Sentry once this gets into `master` to monitor for random EVSS authentication / authorization errors.

Reply:
per Tim Barto: "as long as the gender header is set to acceptable values, of either `MALE` or `FEMALE`, it should not cause an issue for those other EVSS APIs.  that is, though the other APIs will ignore the header (as they do not need the header for their processing), since that header matches back to an original VAAFI header, those other EVSS APIs inherently wind up checking that if the header is set, that it has values of either `MALE` or `FEMALE`"